contrib/http_wasm/filters/http/source/vm_runtime.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
+
+#include <memory>
+
+#include "envoy/common/exception.h"
+#include "envoy/stats/scope.h"
+#include "envoy/stats/stats.h"
+
+#include "source/common/common/logger.h"
+
+#include "absl/strings/str_cat.h"
+#include "contrib/http_wasm/filters/http/source/host/vm_runtime.h"
+#include "contrib/http_wasm/filters/http/source/host/word.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace HttpWasm {
+
+// providing logger to Wasm VM.
+class EnvoyWasmVmIntegration : public Host::WasmVmIntegration, Logger::Loggable<Logger::Id::wasm> {
```

Comment:
I believe this is the abstraction layer that was needed to pass Envoy info to Proxy-Wasm C++ host, so I think it should be possible to remove this entire class